app/components/previews/provider_interface/status_box_component_preview.rb

```diff
@@ -0,0 +1,59 @@
+module ProviderInterface
+  class StatusBoxComponentPreview < ActionView::Component::Preview
+    def awaiting_provider_decision
+      render_component_for ApplicationChoice.find_by(status: :awaiting_provider_decision)
+    end
+
+    def offer
+      render_component_for ApplicationChoice.find_by(status: :offer)
+    end
+
+    def pending_conditions_with_conditions
+      render_component_for ApplicationChoice.find_by(
+        'status = \'pending_conditions\' AND offer IS NOT NULL AND offer->>\'conditions\' != \'[]\'',
+      )
+    end
+
+    def pending_conditions_with_no_conditions
+      render_component_for ApplicationChoice.find_by('status = \'pending_conditions\' AND offer->>\'conditions\' = \'[]\'')
+    end
+
+    def rejected
+      render_component_for ApplicationChoice.find_by(status: :rejected)
```

Comment:
Perhaps this to pick a different one each time?

```suggestion
      render_component_for ApplicationChoice.rejected.order('RANDOM()').first
```

Reply:
This preview renders a `withdrawn_by_us` application on my local setup — should we specify `offer_withdrawn_at` is `nil`?